resources/styles/components/task/breadcrumbs.less

```diff
@@ -0,0 +1,59 @@
+.circle(@radius){
+  font-size: @radius;
+  border-radius: 2 * @radius;
+  line-height: 2 * @radius;
+  min-width: 2 * @radius;
+}
+
+
+// http://caniuse.com/#search=counter
+.task-homework, .task-practice, .task-chapter_practice{
```

Comment:
Can you place separate selectors on separate lines? (and a space before the curly brace)